src/structures/Backup.js

```diff
@@ -0,0 +1,31 @@
+module.exports = class Backup {
+  #api
+  #realm
+  constructor (api, realm, data) {
+    this.#api = api
+    this.#realm = realm
+    Object.assign(this, {
+      id: data.backupId,
+      lastModifiedDate: data.lastModifiedDate,
+      size: data.size,
+      metadata: {
```

Comment:
is there a reason to encapsulate in metadata subfield ?

Reply:
It's as it was returned from the API and think it fits as doesn't relate to the backup but the metadata for the world.